frontend/app/adapters/application.js

```diff
@@ -0,0 +1,11 @@
+import DS from 'ember-data';
+
+export default DS.JSONAPIAdapter.extend({
+  namespace: 'api/v1',
+  pathForType:function(type){
+    type = Ember.String.underscore(type);
+    type = Ember.String.pluralize(type);
+    return type
```

Comment:
Ember has prototype extensions, making it possible to do `type.underscore().pluralize()` immediately